packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/template.html

```diff
@@ -0,0 +1,13 @@
+<oui-message
+    class="mb-3"
+    data-type="info"
+    dismissable
+>
+    <span data-translate="pci_components_vps_announcement_banner_info"></span>
+    <p class="m-0">
+        <a
+            data-translate="pci_components_vps_announcement_banner_info_link"
+            data-ng-href="{{:: $ctrl.tasksInfoLink }}"
```

Comment:
Add `rel="noopener"` as the link is external and also `_blank` to open in new tab ?